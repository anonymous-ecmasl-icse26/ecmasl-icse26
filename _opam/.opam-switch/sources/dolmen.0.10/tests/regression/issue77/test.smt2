(set-logic QF_LIA)
(declare-fun a () Int )
(declare-fun b () Bool )
(assert (let ( (c (ite b 1.0 2.0))) (= 0 (* 2 c))))
(check-sat )
