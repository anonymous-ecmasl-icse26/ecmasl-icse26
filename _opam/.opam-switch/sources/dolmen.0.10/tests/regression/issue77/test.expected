File "tests/regression/issue77/test.smt2", line 4, character 25-28:
4 | (assert (let ( (c (ite b 1.0 2.0))) (= 0 (* 2 c))))
                             ^^^
Error Unbound identifier: `1.0`
Hint: The current logic does not include real arithmtic
